ckanext/xloader/plugin.py

```diff
@@ -107,9 +107,23 @@ def notify(self, entity, operation):
         See: ckan/model/modification.py.DomainObjectModificationExtension
         """
         if operation != DomainObjectOperation.changed \
-        or not isinstance(entity, Resource) \
-        or not getattr(entity, 'url_changed', False):
+        or not isinstance(entity, Resource):
             return
+
+        # If the resource requires validation, stop here if validation
+        # has not been performed or did not succeed. The Validation
+        # extension will call resource_patch and this method should
+        # be called again. However, url_changed will not be in the entity
+        # once Validation does the patch.
+        if _is_validation_plugin_loaded() and \
+        toolkit.asbool(toolkit.config.get('ckanext.xloader.requires_validation')):
```

Comment:
This needs indentation for Flake8.

Reply:
@ThrawnCA lemme know if it is good now on the flake8